tests/common.cpp

```diff
@@ -86,7 +86,7 @@ void TestMaterial::RandomTestSupportFunctionAllObjects()
 
 #ifdef DO_TEST
     for(size_t i=0; i<sObj.size(); i++)
-      supportVector[j*sObj.size() + i] = sObj[i]->support(v);
+      supportVector[j*sObj.size() + i] = sObj[static_cast<unsigned char>(i)]->support(v);
```

Comment:
This is actually an error in sch-core, cf. [this](https://github.com/jrl-umi3218/sch-core/blob/master/include/sch/CD/CD_Scene.h#L49).